docs/management/admin/trusted-apps.asciidoc

```diff
@@ -16,9 +18,11 @@ To add a trusted application:
 
 * `Name your trusted application`: Enter a name for the trusted application.
 
+* `Description`(Optional): Enter a description for the trusted application.
+
 * `Select operating system`: Select the appropriate operating system from the drop-down.
 
-* `Field`: Select the appropriate field you want to use -- `Hash`, `Path`, or (if you are adding a Windows trusted application) `Signature`.
+* `Field`: Select the appropriate field you want to use -- `Hash`, `Path`, or (if you are adding a trusted application on Windows) `Signature`.
 +
 NOTE: You can only add a single field type value per trusted application. For example, if you try to add two `Path` values, you'll get an error message. Hash values must also be valid to add the trusted application. In addition, to minimize the gaps in visibility to the Security solution, be as specific as possible in your entries. For example, combine `Signature` information with known `Paths`.
 +
```

Comment:
Note: the Operator can be `is` or `matches` if field is `path`

Reply:
good catch @dasansol92 

@joepeeples can we add a note here for this distinction?

* For `path` only, users can choose `is` or `matches`.  If users pick `is` then the full path must match exactly.  If users pick `matches` they can include wildcards such as `C:\some_path\*\program.exe`

Reply:
@dasansol92 @kevinlog Thanks for catching this! I noticed `matches` per https://github.com/elastic/security-docs/issues/1243, but I thought that wasn't released until 8.0. But if it's in 7.16, we'll definitely want to add to docs.

Reply:
Thanks @joepeeples - I can see the change below.  I updated the ticket to be marked `7.16` and we can close it with this PR.

https://github.com/elastic/security-docs/issues/1243